mrp_mto_with_stock/__manifest__.py

```diff
@@ -7,9 +8,10 @@
     "summary": "Fix Manufacturing orders to pull from stock until qty is "
                "zero, and then create a procurement for them.",
     "author": "John Walsh, Eficent, Odoo Community Association (OCA)",
+    "maintainer": "Sergio Corato",
```

Comment:
@sergiocorato You should put your github login (`sergiocorato`) here, not your name.

Reply:
done